readthedocs/docsitalia/settings/docker.py

```diff
@@ -30,6 +30,8 @@ class DocsItaliaDockerSettings(CommunityBaseSettings):
     RTD_LATEST_VERBOSE_NAME = RTD_LATEST
     RTD_STABLE = 'stabile'
     RTD_STABLE_VERBOSE_NAME = RTD_STABLE
```

Comment:
Domanda scema: ma questo `_VERBOSE_NAME` servono? E, se sì, devono avere anche loro versioni inglesi?

Reply:
Si _VERBOSE_NAME servono ma per operare con i valori da DB. Noi non mettiamo versione inglese in DB, soltanto nel URL, per questo non ci serve versione inglese per Verbose Name